src/vec-alloc.md

```diff
@@ -157,7 +155,11 @@ such we will guard against this case explicitly.
 Ok with all the nonsense out of the way, let's actually allocate some memory:
 
 ```rust,ignore
-use std::alloc::oom;
+#![feature(allocator_api, ptr_internals)]
```

Comment:
I'm pretty sure you want to just use the stable free functions in std::alloc, which should avoid the need for any features?

https://doc.rust-lang.org/std/alloc/index.html

Reply:
I may have overlooked. Will use stable API instead.